src/bluesearch/database/topic.py

```diff
@@ -297,6 +297,88 @@ def get_topics_for_pmc_article(
     return journal_topics
 
 
+def get_topics_for_arxiv_articles(
+    arxiv_paths: Iterable[pathlib.Path] | Iterable[str], batch_size: int = 400
```

Comment:
Just wondering about the `arxiv_paths` type...Would it not be possible to do `Iterable[pathlib.Path | str]`? If yes, not sure whether which one is the recommended one

Reply:
As @Stannislav pointed out, they mean different things and the one you are suggesting covers more cases.
But luckily for us either type would work in this case, and I like your proposal because it's more compact.
Fixed in b063949.

Reply:
True, they mean something different:) Thanks!